### ISSUE TITLE
Generate Dotnet Core Project

### ISSUE THREAD

**Issue by adrianoc** [adrianoc/cecilifier#7]
This is an improvement over #5 idea.

We could generate the project structure (alongside required files) and return as a zip file ready to be built in dotnet core; all the user would need to do would be to extract the file contents and open a console (or open the project in Visual Studio, Rider, etc).

Maybe we can use a trick like this (works on a page loaded from local computer; not sure this will work on a page loaded from a webserver though):

```html
<html>
<body>
<button
     type="button" 
     onclick="callRESTServiceThatReturnsAZipStream()'">Download</button>
</body>
</html>
```

- [ ] Add option to page ([maybe something like this](https://www.w3schools.com/code/tryit.asp?filename=G5MGILJCHRYL))
- [x] Figure out list of files required and its contents
- [x] Add REST endpoint

**Comment by adrianoc** [comment]
Basically we need to generate:

- Project file (csproj)
```
 <Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp3.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Mono.Cecil" Version="0.11.0" />
  </ItemGroup>
</Project>
```

- Cecilified File (.cs)

- TypeHelpers.cs

**Comment by adrianoc** [comment]
implemented at ab86d8bdfd405cb629f6976a6ebb1f587b076851

Decided to not introduce options. Instead just introduced a button
